Changelog.md

```diff
@@ -1,9 +1,21 @@
 Changelog
 =========
 
-### 2.1.3 (2018-xx-xx)
+### 2.2.0 (2020-01-08)
 
 * Added Mongolian translation.
+* Added an email prodiver.
+* Added a custom user checker.
+* Added PHP 7.3 support.
```

Comment:
Could we provide PHP 7.4 support too ?

Reply:
https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2955